src/atoms/datetime-picker/datetime-picker.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState, useCallback } from "react";
+import { storiesOf } from "@storybook/react";
+import { DatetimePicker } from "./datetime-picker";
+import { action } from "@storybook/addon-actions";
+
+const ControlledDateTimePicker = () => {
```

Comment:
Something that inherently holds state is normally "uncontrolled", so this would be "UncontrolledDateTimePicker".